polkadot/runtime/parachains/src/lib.rs

```diff
@@ -71,8 +79,6 @@ pub trait FeeTracker {
 	/// The factor that is used to increase the current message fee factor for every sent kilobyte.
 	const MESSAGE_SIZE_FEE_BASE: FixedU128 = FixedU128::from_rational(1, 1000); // 0.001
 
-	fn get_min_fee_factor() -> FixedU128;
```

Comment:
@serban300 can we just do a 1 as default impl?
```
fn get_min_fee_factor() -> FixedU128 {
    MIN_FEE_FACTOR
}
```

This way, no implementation needs implement it and on the other side, it will still have possibility to change it, if needed
And also we don't need to propage MIN_FEE_FACTOR constant anywhere, so we can use `Self::get_min_fee_factor()` instead

Reply:
I also thought about this. Actually that was my first implementation, but then I discarded it. 

We could, but I don't know if it's worth it. I would prefer to avoid over complicating the implementation if it's not needed. Right now the min fee factor is 1 everywhere and personally I can't think of a use-case where we would need it to be more than 1. I think that if we want to increase the min fee, we can just increase the base fee directly. WDYT ?

Reply:
Well, if we could use `Self::get_min_fee_factor` (with the default `MIN_FEE_FACTOR`) instead of spreading the `MIN_FEE_FACTOR` constant across the whole codebase, I don't think it's overcomplicating or overengineering. It also gives us the option to override it per implementation - which is very unlikely, but maybe there could be some strange use case where a delivery wants to apply lower fees than 1 :D or maybe you want to ignore delivery fees by setting the fee-factor to 0)

Reply:
Ok, done. PTAL !

Reply:
cool, nice :)